lib/truffle/truffle/cext.rb

```diff
@@ -1894,14 +1894,18 @@ def rb_tr_writable(mode)
     mode == File::Constants::WRONLY || mode == File::Constants::RDWR
   end
 
+  def rb_backref_get
+    Truffle::RegexpOperations.last_match(Truffle::ThreadOperations.ruby_caller(3))
```

Comment:
Hard-coded depth is of course far from ideal.
But we already have this problem in the C-ext impl elsewhere.
Maybe we need a primitive to get the binding of a user frame, and ignore all core/`cext.rb` frames?